stunnel/stunnel.ml

```diff
@@ -115,7 +115,14 @@ type t = { mutable pid: pid; fd: Unixfd.t; host: string; port: int;
            verified: bool;
          }
 
-let config_file verify_cert extended_diagnosis host port =
+let debug_conf_of_bool verbose : string =
+  if verbose then "debug=authpriv.7" else "debug=authpriv.5"
+
+let debug_conf_of_env () : string =
```

Comment:
I understand this is a backport and we should not change it. But I think this function should return a boolean and whoever needs something else should do the conversion on the client side. This feel overspecialized to me.

Reply:
Yes, I know your point, Here we just want to simply the caller to retrieve the config from environment directly. 
Anyway,  Let's just keep it as it is backport, otherwise, we may need to re-work on master.